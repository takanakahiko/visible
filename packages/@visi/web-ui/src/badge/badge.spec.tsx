import React from 'react';

import { render } from '../render';
import { Badge } from '.';

describe('Badge', () => {
  it('matches snapshot', () => {
    const { container } = render(<Badge variant="red" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
