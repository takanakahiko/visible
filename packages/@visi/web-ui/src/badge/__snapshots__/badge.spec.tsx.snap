// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge matches snapshot 1`] = `
.c0 {
  width: -webkit-min-content;
  width: -moz-min-content;
  width: min-content;
  padding: 2px 8px;
  border-radius: 4px;
  background-color: #d32626;
  color: white;
  font-size: 12px;
  font-weight: bolder;
}

<div
  class="c0"
/>
`;
